src/filters/filter-fields-component.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.filters.component:pfFilterFields
+ *
```

Comment:
I believe should this have a restrict E here to have the docs show the component correctly in the example code (instead of <ANY ...